django_prometheus/cache/backends/django_memcached_consul.py

```diff
@@ -17,4 +17,4 @@ def get(self, key, default=None, version=None):
         else:
             django_cache_misses_total.labels(
                 backend='django_memcached_consul').inc()
-        return cached
+        return cached or default
```

Comment:
For my own curiosity: how can we reach this since we're passing the default to `super().get()`? `cached` should already be equal to `default` if the value didn't exist, shouldn't it?

Reply:
But we don't use default in super().get():
        cached = super(MemcachedCache, self).get(
            key, default=None, version=None)
In current code we never return default.

Reply:
For me problem was in:
self.cache.get(self.key, []) should be [] but get None

Reply:
And we can't send default in to super.
cache = {
    'a': 'default'
}
cache.get('a') is 'default' and cache.get('a', 'default') is 'default'. In this case we can't detect is it cache_miss or cache_hit. And we should check default in prometheus wrapper. That's why is think this solution is correct )

Reply:
Oh, duh :D thanks for the explanation, I was clearly not thinking right.

Reply:
thanks for your work 8)